tests/framework/base/WidgetTest.php

```diff
@@ -46,6 +48,27 @@ public function testBeginEnd()
         $this->assertSame('<run-test>', $output);
     }
 
+    public function testDI()
```

Comment:
Add comment
~~~php
/**
 * @covers Widget::begin
 * @covers Widget::end
 */
~~~
https://phpunit.readthedocs.io/en/9.5/annotations.html#covers

Reply:
Thanks for suggestion i added comments